### ISSUE TITLE
is there way to generate single rerun.txt file after the parallel test is completed

### ISSUE THREAD

**Issue by mmuntakim15** [temyers/cucumber-jvm-parallel-plugin#65]
Hi, Is there way to generate single rerun.txt file after the parallel tests are completed so we can just rerun all failed tests as post single thread instead rerun it in parallel mode. For example. currently when we run parallel tests using this plugin, we don't have the capability to add "rerun:rerun.txt" plugin to all the runners. I know that [this](https://github.com/temyers/cucumber-jvm-parallel-plugin/pull/30) pull request is open related to rerun functionality. Please kindly let us know if this pull request will fix our issue. Btw thanks for building this plugin, It's been helping us everyday with our test execution. 

Thanks 
Mohammad

**Comment by sugatmankar** [comment]
Hello @mmuntakim15, Thanks for appreciation.

I had this changes but removed for purpose "tagwise parallel re-runner".

Please try steps mentioned in comments https://github.com/temyers/cucumber-jvm-parallel-plugin/pull/53.

If you still face issue let us know, we will fix it.

**Comment by mmuntakim15** [comment]
Hi @sugatmankar thanks for your reply. I understand that in version 2.0.0 we will have the parallel rerun functionality, My question was more toward is there rerun functionality exists in current version 1.2.1, which we are using currently, please let me know if you need further details.

**Comment by temyers** [comment]
At the moment I don't think it will work in 1.2.1, since the rerun.txt file would probably be corrupted/overwritten when the Cucumber runners are running in parallel.

I haven't tested it, but you could try using `<format>rerun:rerun.txt</format>` and see how you go.

2.0.0 will provide proper support for rerunning and should be ready before too long.

**Comment by temyers** [comment]
2.0.1 is now released supporting parallel running of scenarios.  
Re-running is not currently supported whilst I investigate some other options.

**Comment by temyers** [comment]
Duplicate of issue #31

**Comment by adarshkv** [comment]
Hi All,

I have set up a Selenium Java (Cucumber) framework for my project and I am stuck in the same issue. I want to generate a single rerun file having all the failed test cases/ scenarios in them after performing parallel execution, how can the same be achieved ?

I am using below versions:
cucumber-jvm-parallel-plugin - 4.2.0
maven-surefire-plugin - 2.20.1

Can you please help me out on this?

Thanks 
Adarsh
